src/utils/initiative-utils.ts

```diff
@@ -203,18 +203,54 @@ export class InitiativeBuilder {
 		}
 		turnText += '\n';
 		if (actorsInGroup.length === 1 && actorsInGroup[0].name === actorGroup.name) {
-			//We're just displaying the actor in its initiative slot
+			//We're displaying the actor in its initiative slot with HP, Max HP, SP and Max SP shown if present
```

Comment:
This comment was really just meant to define the difference between a single actor in a group vs multiple actors in a group. This distinction isn't important now, but it will be in the future for familiars and such